src/Module/Conversation/Channel.php

```diff
@@ -0,0 +1,398 @@
+				'url'       => 'channel/' . self::AUDIO,
+				'sel'       => self::$content == self::AUDIO ? 'active' : '',
+				'title'     => DI::l10n()->t('Posts with audio'),
+				'id'        => 'channel-audio-tab',
+				'accesskey' => 'a'
+			];
+
+			$tab_tpl = Renderer::getMarkupTemplate('common_tabs.tpl');
+			$o .= Renderer::replaceMacros($tab_tpl, ['$tabs' => $tabs]);
+
+			Nav::setSelected('channel');
+
+			DI::page()['aside'] .= Widget::accountTypes('channel/' . self::$content, self::$accountTypeString);
+
+			if (!in_array(self::$content, [self::FOLLOWERS, self::FORYOU]) && DI::config()->get('system', 'community_no_sharer')) {
+				$path = self::$content;
+				if (!empty($this->parameters['accounttype'])) {
+					$path .= '/' . $this->parameters['accounttype'];
+				}
+				$query_parameters = [];
+
+				if (!empty($_GET['min_id'])) {
+					$query_parameters['min_id'] = $_GET['min_id'];
+				}
+				if (!empty($_GET['max_id'])) {
+					$query_parameters['max_id'] = $_GET['max_id'];
+				}
+				if (!empty($_GET['last_created'])) {
+					$query_parameters['max_id'] = $_GET['last_created'];
+				}
+
+				$path_all       = $path . (!empty($query_parameters) ? '?' . http_build_query($query_parameters) : '');
+				$path_no_sharer = $path . '?' . http_build_query(array_merge($query_parameters, ['no_sharer' => true]));
+				DI::page()['aside'] .= Renderer::replaceMacros(Renderer::getMarkupTemplate('widget/community_sharer.tpl'), [
+					'$title'           => DI::l10n()->t('Own Contacts'),
+					'$path_all'        => $path_all,
+					'$path_no_sharer'  => $path_no_sharer,
+					'$no_sharer'       => !empty($_REQUEST['no_sharer']),
+					'$all'             => DI::l10n()->t('Include'),
+					'$no_sharer_label' => DI::l10n()->t('Hide'),
+					'$base'            => 'channel',
+				]);
+			}
+
+			if (Feature::isEnabled(DI::userSession()->getLocalUserId(), 'trending_tags')) {
+				DI::page()['aside'] .= TrendingTags::getHTML(self::$content);
+			}
+
+			// We need the editor here to be able to reshare an item.
+			$o .= DI::conversation()->statusEditor([], 0, true);
+		}
+
+		$items = self::getItems();
+
+		if (!DBA::isResult($items)) {
+			DI::sysmsg()->addNotice(DI::l10n()->t('No results.'));
+			return $o;
+		}
+
+		$o .= DI::conversation()->render($items, Conversation::MODE_CHANNEL, false, false, 'created', DI::userSession()->getLocalUserId());
+
+		$pager = new BoundariesPager(
+			DI::l10n(),
+			DI::args()->getQueryString(),
+			$items[0]['created'],
+			$items[count($items) - 1]['created'],
+			self::$itemsPerPage
+		);
+
+		if (DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'infinite_scroll')) {
+			$o .= HTML::scrollLoader();
+		} else {
+			$o .= $pager->renderMinimal(count($items));
+		}
+
+		return $o;
+	}
+
+	/**
+	 * Computes module parameters from the request and local configuration
+	 *
+	 * @throws HTTPException\BadRequestException
+	 * @throws HTTPException\ForbiddenException
+	 */
+	protected function parseRequest()
```

Comment:
Please a `$request` parameter to this method instead of querying `$_GET` directly.

Reply:
Done